client/src/sections/Login/Login.tsx

```diff
@@ -1,17 +1,27 @@
-import React from 'react';
+import React, {useState, useCallback} from 'react';
 import {useHistory} from 'react-router-dom';
 import {Button, Card, Heading, ButtonGroup} from '@shopify/polaris';
-import logo from 'icons/logo.jpg';
+import logo from 'icons/logo.png';
 
 import './Login.css';
+import LoginModal from './LoginModal';
 
 export default function Login() {
   const history = useHistory();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [active, setActive] = useState(false);
+
+  const handleChange = useCallback(() => setActive(!active), [active]);
 
   const handleLogin = () => {
     history.push('/home');
```

Comment:
Not to be done in this PR, but this will need to change the user state once it is set up in redux.